src/numerical_flux/entropy_cons_numerical_flux.cpp

```diff
@@ -0,0 +1,70 @@
+#include "entropy_cons_numerical_flux.hpp"
+#include "ADTypes.hpp"
+
+namespace PHiLiP {
+namespace NumericalFlux {
+
+template <int dim, int nstate, typename real>
+std::array<real, nstate> EntropyConsNumFlux<dim,nstate,real>::evaluate_flux(
+ const std::array<real, nstate> &soln_int,
+    const std::array<real, nstate> &soln_ext,
+    const dealii::Tensor<1,dim,real> &normal_int) const
+{
+//    using PDE_enum = Parameters::AllParameters::PartialDifferentialEquation;
+//    assert(pde_physics->parameters->pde_type == PDE_enum::burgers_inviscid);
+
+    using RealArrayVector = std::array<dealii::Tensor<1,dim,real>,nstate>;
+    RealArrayVector conv_phys_flux_int;
+    RealArrayVector conv_phys_flux_ext;
+
+    conv_phys_flux_int = pde_physics->convective_flux (soln_int);
+    conv_phys_flux_ext = pde_physics->convective_flux (soln_ext);
+    
+    RealArrayVector flux_avg;
+    for (int s=0; s<nstate; s++) {
+        flux_avg[s] = 0.0;
+        for (int d=0; d<dim; ++d) {
+            flux_avg[s][d] = 0.5*(conv_phys_flux_int[s][d] + conv_phys_flux_ext[s][d]);
+        }
+    }
```

Comment:
Previous comment still holds. Why not use the function array_average?

Reply:
fixed as per issue #125